didyoumean/__init__.py

```diff
@@ -1,2 +1,3 @@
 """Empty file. Might grow in the future."""
-import didyoumean_api
+from didyoumean import didyoumean_api
+from didyoumean.didyoumean_internal import add_suggestions_to_exception
```

Comment:
pep8 complains that:
didyoumean/__init__.py:3:72: W292 no newline at end of file